index.js

```diff
@@ -115,8 +117,10 @@ class CacclDeployCommander extends Command {
    * @param {string} appName
    */
   getAppPrefix(appName) {
-    if (this.ssmRootPrefix === undefined
-        || (this.app === undefined && appName === undefined)) {
+    if (
+      this.ssmRootPrefix === undefined ||
```

Comment:
Aww I liked your other formatting better. I always vote for operators coming first:

```
if (
  something
  && something else
  && another thing
) {

}